src/vis/vis.js

```diff
@@ -21,7 +20,6 @@ var STATE_ERROR = 'error'; // vis has been sent to Windshaft and there were some
 var VisModel = Backbone.Model.extend({
   defaults: {
     loading: false,
-    https: false,
```

Comment:
In theory, `https` could be undefined. According to our actual docs:

- `http === true`. Will try to transform **http** urls of basemaps to **https**
- `https === false`. Will try to tranform **https** urls of basemaps to **http**
- `https === undefined`. Will leave basemap urls as they are.

The truth is that, right now the vizjson files don't include `https` and we're not using the `https` option either. So, `https` is being set `true` if the page is served over `https` and `false` otherwise.